tests/integration.sh

```diff
@@ -1265,13 +1281,16 @@ function test_transfer_from() {
 function test_send_from() {
     set -e
     local contract_addr="$1"
+    local skip_register_receiver="$2"
 
     log_test_header
 
-    local receiver_addr
-    receiver_addr="$(create_receiver_contract)"
-    # receiver_addr='secret17k8qt6aqd7eee3fawmtvy4vu6teqx8d7mdm49x'
-    register_receiver "$receiver_addr" "$contract_addr"
+    local receiver_addr="$(create_receiver_contract)"
+    local receiver_hash="$(secretcli q compute contract-hash $receiver_addr | sed 's/^0x//')"
+
+    if [ "$skip_register_receiver" != "true" ]; then
+        register_receiver "$receiver_addr" "$contract_addr"
+    fi
```

Comment:
same as above, please separate declaration from assignment of command output.

Reply:
Done